spec/fixtures/activities/ams/gcrf/gcrf_real_and_complete_legacy_file.xml

```diff
@@ -0,0 +1,1460 @@
+
```

Comment:
If this file isn't used as part of the Newton ingest I wonder if it should wait to be included in the work that makes the GCRF ingest possible, keeping the related changes together.

Reply:
Gone!